ccmlib/common.py

```diff
@@ -76,6 +77,36 @@ class UnavailableSocketError(CCMError):
     pass
 
 
+class LogPatternToVersion(object):
+    def __init__(self, patterns_to_versions):
+        assert type(patterns_to_versions) is dict, "LogPatternToVersion takes a dictionary."
```

Comment:
You would do this with a `isinstance` call, but I don't think this is a use case to do type checking for. Just let it blow up if the passed-in thing doesn't work.